halo2_proofs/src/poly/commitment/prover.rs

```diff
@@ -60,27 +60,29 @@ pub fn create_proof<
     // Challenge that will ensure that the prover cannot change P but can only
     // witness a random polynomial commitment that agrees with P at x, with high
     // probability.
-    let iota = *transcript.squeeze_challenge_scalar::<()>();
+    let xi = *transcript.squeeze_challenge_scalar::<()>();
 
     // Challenge that ensures that the prover did not interfere with the U term
     // in their commitments.
     let z = *transcript.squeeze_challenge_scalar::<()>();
 
-    // We'll be opening `s_poly_commitment * iota + P - [v] G_0` to ensure it
-    // has a root at zero.
-    let mut final_poly = s_poly * iota + px;
-    let v = eval_polynomial(&final_poly, x);
-    final_poly[0] = final_poly[0] - &v;
-    let blind = s_poly_blind * Blind(iota) + blind;
-    let mut blind = blind.0;
+    // We'll be opening `P' = P - [v] G_0 + [\xi] S` to ensure it has a root at
+    // zero.
+    let mut p_prime_poly = s_poly * xi + p_poly;
```

Comment:
```suggestion
    // We'll be opening `P' = P - [v] G_0 + [ξ] S` to ensure it has a root at
    // zero.
    let mut p_prime_poly = s_poly * ξ + p_poly;
```

Reply:
Non-ASCII identifiers was stabilised in 1.53: https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#unicode-identifiers

Once #482 is closed, we can make this rename.